### PR TITLE
winapi: Fix XBE file header

### LINKED ISSUE #493: XBE header magic too small
https://github.com/XboxDev/nxdk/blob/a7a5ecff1d5b6b6af187c4de013ddf26509851b5/lib/winapi/winnt.h#L286

The value of this one is `0x48454258` on retail titles, a full DWORD long, not just WORD.

### PR DESCRIPTION
**Pull request by dracc**
Closes #493

### CODE REVIEW

File: lib/winapi/winnt.h [hunk 1]
```diff
@@ -283,8 +283,8 @@ typedef struct _XBE_SECTION_HEADER
 
 typedef struct _XBE_FILE_HEADER
 {
-    WORD Magic;
-    UCHAR Signature[256];
+    DWORD Magic;
+    UCHAR Signature[255];
```
Comment (thrimbor) [inline lib/winapi/winnt.h:287]: This is not correct, the signature is 256 bytes, not 255. You can compare with cxbe and xboxdevwiki.

**Review by thrimbor — COMMENTED** [review]
Good catch on the typo. Can't believe I missed that when testing (got screwed by padding I guess).

File: lib/winapi/winnt.h (hunk 1 above)
Comment (dracc) [reply lib/winapi/winnt.h]: My testing suggested this was correct. Thrimbor tested better, showing the opposite and that my testing simply gave false positives.

**Comment by thrimbor** [comment]
Looks good, thanks for fixing!

**Pull request merged**